<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="liangdeMacBook-Pro.local" name="com.young.appiumcombat.testcases.android.applytax.Android_ApplyTaxPage_001_UiCheck_Test" tests="4" failures="1" timestamp="24 May 2017 10:26:17 GMT" time="41.607" errors="0">
  <testcase name="Case_004_backToHomePage" time="13.149" classname="com.young.appiumcombat.testcases.android.applytax.Android_ApplyTaxPage_001_UiCheck_Test"/>
  <testcase name="Case_001_intoApplyTax" time="7.739" classname="com.young.appiumcombat.testcases.android.applytax.Android_ApplyTaxPage_001_UiCheck_Test"/>
  <testcase name="Case_002_applyTaxPageUICheck" time="14.519" classname="com.young.appiumcombat.testcases.android.applytax.Android_ApplyTaxPage_001_UiCheck_Test"/>
  <testcase name="Case_003_wantToUseThisFunction" time="6.200" classname="com.young.appiumcombat.testcases.android.applytax.Android_ApplyTaxPage_001_UiCheck_Test">
    <failure type="java.lang.AssertionError" message="经检查：‘我也想用此功能’点击失效 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 经检查：‘我也想用此功能’点击失效 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.young.appiumcombat.testcases.android.applytax.Android_ApplyTaxPage_001_UiCheck_Test.Case_003_wantToUseThisFunction(Android_ApplyTaxPage_001_UiCheck_Test.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- Case_003_wantToUseThisFunction -->
</testsuite> <!-- com.young.appiumcombat.testcases.android.applytax.Android_ApplyTaxPage_001_UiCheck_Test -->
